js/ui/components/clubhouse.js

```diff
@@ -114,12 +115,31 @@ var ClubhouseNotificationSource = new Lang.Class({
     },
 });
 
+var ClubhouseOpenButton = new Lang.Class({
+    Name: 'ClubhouseOpenButton',
+    Extends: St.Button,
+
+    _init: function(params) {
+        this.parent(params);
+
+        this._icon = new St.Icon({ style_class: 'clubhouse-open-button-icon' });
+        this.child = this._icon;
+    },
+
+    setHighlighted: function(highlighted=true) {
+        if (highlighted) {
+            this._icon.add_style_pseudo_class('highlighted');
+        } else {
```

Comment:
Nitpick: Don't use curly brackets for one-line bodies.

Reply:
Yeah, it seems the Shell follows that style. So I'll adjust.